specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -53,7 +53,12 @@ model BingGroundingToolDefinition extends ToolDefinition {
   type: "bing_grounding";
 
   @doc("The list of connections used by the bing grounding tool.")
-  bing_grounding: ToolConnectionList;
+  bing_grounding: {
+    @removed(Versions.`2025-05-15-preview`)
```

Comment:
Should this be removed in 2025-05-01 instead?

Reply:
@joheredi If you can point me to a reference to 2025-05-01 in the Versions namespace, I'll use that instead.

Reply:
Yeah I think we need to add 2025-05-01 here: https://github.com/Azure/azure-rest-api-specs/blob/51e284322b1c68911e510fbbb68c934f719ad76f/specification/ai/Azure.AI.Agents/main.tsp#L52

AFAIK, the versions for GA will be:

"2025-05-01"
"2025-05-15-preview"
"v1"

Not sure if TypeSpec supports the mixed scheme of date and v1 yet, but this is something that the TypeSpec team should be supporting soon.

Reply:
@dargilco @joheredi should I add the 2025-05-01 in this PR or add the version and update this reference in a future PR?

Reply:
[Updated](https://github.com/Azure/azure-rest-api-specs/pull/34343/commits/0bd23a6a62a1111f0736fbf87d24f5197f8bc503)